tests/integration/integration_test_Entity.py

```diff
@@ -141,7 +141,7 @@ def test_Entity():
 
     newfolder = Folder('Testing Folder', parent=project)
     newfolder = syn.store(newfolder)
-    link = Link(newfolder.id, parent = folder.id)
```

Comment:
If you notice https://github.com/Sage-Bionetworks/synapsePythonClient/pull/420/files/83281f034455f150013c0de58177b6f89a09f3dd#diff-7c91124835c610ac5387325cc1920f6dR123

I do a link out to a file entity by using the id.  I passed in the newfolder entity to test the feature I added in the Link entity.